src/app/demos/route-metadata/route-order-test/j/index.html

```diff
@@ -0,0 +1,5 @@
+<stache
+  pageTitle="j"
+  navOrder="0">
+   0
```

Comment:
Correct,  we are not accepting 0 as a valid order.   In this case we are not 0 indexing our 'ordering'.   Our client base would expect to put a `1`  and it be the first item in the nav and a `4` to be the 4th, not off by one as would occur with a 0 indexed version.